### PR TITLE
Serial tests

### PR DESCRIPTION
**Pull request by dcermak**
This PR adds the `serial` mark to pytest so that certain tests can be executed in serial.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,7 +14,8 @@ commands =
     m8a-fetch-language-containers {envname}
     # -n auto tells pytest to use as many processors as detected
     # to run all the tests from the module test_{envname} in parallel
-    pytest -n auto test_{envname}.py --junitxml={toxinidir}/junit.xml []
+    pytest -n auto -m "not serial" test_{envname}.py --junitxml={toxinidir}/junit_parallel.xml []
+    pytest -n0 -m "serial" test_{envname}.py --junitxml={toxinidir}/junit_serial.xml []
```
Comment (evrardjp) [inline tox.ini]: You can remove -n0 . I think it's clearer, and that way the pytest-xdist isn't involved at all, iirc.

**Review by evrardjp — APPROVED** [review]
I like this, it's clear.

File: tox.ini (hunk 1 above)
Comment (evrardjp) [reply tox.ini]: Meh, let's ignore this ^

**Comment by evrardjp** [comment]
Waiting for node to return green beffore pressign the big green button.

**Review by evrardjp — CHANGES_REQUESTED** [review]
I would prefer if we update the commit message for the add pytest serial mark.

**Review by evrardjp — DISMISSED** [review]
LGTM

**Comment by dcermak** [comment]
The react.js tests are consistently failing in github actions, I'd drop them because they don't add any value imho. WDYT @evrardjp?

**Comment by dcermak** [comment]
Jean-Philippe Evrard ***@***.***> writes:

> Yeah let's remove them. We have enough examples. 

I already removed them.

**Pull request merged**